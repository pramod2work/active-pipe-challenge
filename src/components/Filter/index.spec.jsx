import React from 'react'
import { shallow } from 'enzyme'

import { Filter } from './index'

describe('Filter Component test', () => {
  test('Component Snapshot Testing', () => {
    const component = shallow(<Filter />)
    expect(component).toMatchSnapshot()
  })
})