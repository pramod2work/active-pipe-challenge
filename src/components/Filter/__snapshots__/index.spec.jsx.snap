// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Filter Component test Component Snapshot Testing 1`] = `
<div
  className="filter"
>
  <label
    htmlFor="filterProperty"
  >
    Filter Status
  </label>
  <select
    id="filterProperty"
    name="filterProperty"
    onChange={[Function]}
  >
    <option
      value=""
    >
      All
    </option>
    <option
      key="current"
      value="current"
    >
      Current
    </option>
    <option
      key="sold"
      value="sold"
    >
      Sold
    </option>
    <option
      key="off_market"
      value="off_market"
    >
      Off Market
    </option>
    <option
      key="withdran"
      value="withdran"
    >
      Withdrawn
    </option>
  </select>
</div>
`;
